torch/distributions/multivariate_normal.py

```diff
@@ -90,36 +100,47 @@ class MultivariateNormal(Distribution):
     Args:
         loc (Tensor): mean of the distribution
         covariance_matrix (Tensor): positive-definite covariance matrix
+        precision_matrix (Tensor): positive-definite precision matrix
         scale_tril (Tensor): lower-triangular factor of covariance, with positive-valued diagonal
 
     Note:
-        Only one of `covariance_matrix` or `scale_tril` can be specified.
+        Only one of :attr:`covariance_matrix` or :attr:`precision_matrix` or
+        :attr:`scale_tril` can be specified.
 
-        Using `scale_tril` will be more efficient: all computations internally
-        are based on `scale_tril`. If `covariance_matrix` is passed instead,
-        this is only used to compute the corresponding lower triangular
-        matrices using a Cholesky decomposition.
+        Using :attr:`scale_tril` will be more efficient: all computations internally
+        are based on :attr:`scale_tril`. If :attr:`covariance_matrix` or
+        :attr:`precision_matrix` is passed instead, it is only used to compute
+        the corresponding lower triangular matrices using a Cholesky decomposition.
     """
     arg_constraints = {'loc': constraints.real_vector,
                        'covariance_matrix': constraints.positive_definite,
```

Comment:
Could you also add a constraint `'precision_matrix': constraints.positive_definitie`?

Reply:
now added :)